### ISSUE TITLE
Review: Use of ? : when not necessary?

### ISSUE THREAD

**Issue by waded** [klaethecoder/JavaScript-Practice#1]
On this line (and a few others)

https://github.com/klaethecoder/JavaScript-Practice/blob/d211d5c343e172a9f0d496ff87ea9b9f4fd9bdcb/palindrome.js#L5

is the `? true: false` structure necessary? (What does the equality expression to the left of the `?` result in?)

**Comment by klaethecoder** [comment]
> On this line (and a few others)
> 
> [JavaScript-Practice/palindrome.js](https://github.com/klaethecoder/JavaScript-Practice/blob/d211d5c343e172a9f0d496ff87ea9b9f4fd9bdcb/palindrome.js#L5)
> 
> Line 5 in [d211d5c](/klaethecoder/JavaScript-Practice/commit/d211d5c343e172a9f0d496ff87ea9b9f4fd9bdcb)
> 
>  return [...word.toLowerCase()].reverse().join('') == word.toLowerCase() ? true : false 
> is the `? true: false` structure necessary? (What does the equality expression to the left of the `?` result in?)

The expression to the left of the '?' needed for the ternary operator. 

condition ? expression if true : expression if false

Here is the link to the MDN reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

**Comment by waded** [comment]
Yes, but do you need to use ternary here? Would the following have the same result?

```
function palindrome(word) {
    return [...word.toLowerCase()].reverse().join('') == word.toLowerCase()
}
```

**Comment by klaethecoder** [comment]
I see that makes more sense. I think I learned a new thing and was trying to use it correctly all the time without really looking at refactoring the code. Thank you for pointing this out.
